### PR TITLE
Homework4: Stepan Balakir

### PR DESCRIPTION
**Pull request by balstepan**
### PR checklist
Please make sure you've passed PR checklist
- [x] my PR passes CI checks (flake8 and pycodestyle)
- [x] my PR named according to guidelines.
- [x] my PR contains complete tasks descriptions for all tasks
- [x] my PR made from the fork feature branch to maxim homework# branch.
- [x] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore

### CODE REVIEW

File: src/task2.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+"""
+
+
+def country_of_city(str_):
+    list_of_input = str_.split('\n')
+    countries_cities = {}
+
+    for string in list_of_input[1:int(list_of_input[0]) + 1]:
+        list_of_country_and_its_cities = string.split()
+        country = list_of_country_and_its_cities[0]
```
Comment (belovmd) [inline src/task2.py]: country, *cities = string.split()

File: src/task2.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        cities = list_of_country_and_its_cities[1:]
+        countries_cities[country] = cities
+
```
Comment (belovmd) [inline src/task2.py:40]: {city: [list of countries]} would significantly simplify your code

File: src/task4.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+"""
+Даны два списка чисел. Посчитайте, сколько различных чисел входит только в один
+из этих списков
+"""
+
+
+def numbers_in_only_one_list(lst1, lst2):
+    return len(set(lst1) & set(lst2))
```
Comment (belovmd) [inline src/task4.py]: your solution doesn't solve task in description

File: src/task6.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+содержится в этом тексте.
+"""
+
+
+def count_words(str_):
+    count = 0
+    list_of_strings = str_.split('\n')
+    for string in list_of_strings:
+        count += len(string.split())
+    return count
```
Comment (belovmd) [inline src/task6.py]: `return len(set(str_.split()))`

File: src/task7.py [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+
+big = 99
+small = 8
+
+while 1:
+    if not big % small:
+        break
+    else:
+        big, small = small, big % small
```
Comment (belovmd) [inline src/task7.py]: weird naming ))

**Review by belovmd — CHANGES_REQUESTED** [review]
Please take a look at this minor issues

**Review by balstepan — COMMENTED** [review]
All issues are closed.

**Review by belovmd — COMMENTED** [review]
9

**Pull request closed without merging**